public/app/features/dashboard/components/PanelEditor/state/reducers.ts

```diff
@@ -9,7 +9,7 @@ export const PANEL_EDITOR_UI_STATE_STORAGE_KEY = 'grafana.dashboard.editor.ui';
 
 export const DEFAULT_PANEL_EDITOR_UI_STATE: PanelEditorUIState = {
   isPanelOptionsVisible: true,
-  rightPaneSize: 350,
+  rightPaneSize: 400,
```

Comment:
@dprokop as I understand this change means users will always see tabs as deafult?

Reply:
Yes